README.md

```diff
@@ -38,13 +37,79 @@ helm install lfx-platform ./charts/lfx-platform -n lfx -f charts/lfx-platform/va
 
 ## Components
 
-The LFX v2 Platform includes the following core components:
-
-- **Traefik**: API Gateway and Ingress Controller
-- **OpenFGA**: Fine-Grained Authorization with Relationship-Based Access Control (ReBAC)
-- **Heimdall**: Identity and Access Proxy
-- **NATS**: Messaging and event streaming system
-- **OpenSearch**: Search and analytics engine
+LFX v2 includes the following infrastructure components:
+
+- **Traefik**: API Gateway and Ingress Controller.
+- **OpenFGA**: Fine-Grained Authorization with Relationship-Based Access
+  Control (ReBAC).
+- **Heimdall**: Access decision service, bridges Traefik to OpenFGA.
+- **NATS**: Messaging layer used by LFX v2 resource APIs to communicate with
+  each other and with platform components.
+- **OpenSearch**: Powers platform global search and audit log capabilities.
+
+Building on those, custom platform components provide shared services essential
+to the LFX v2 platform:
+
+- **indexer**: Processes messages from resource APIs to keep OpenSearch in sync
```

Comment:
Can link to the [indexer service repo](https://github.com/linuxfoundation/lfx-v2-indexer-service)